Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibsAPI.cs

```diff
@@ -0,0 +1,1158 @@
+        /// <param name="projectName">VBA Project name, as declared in the VBE</param>
+        /// <param name="className">Document class name, as declared in the VBA project</param>
+        /// <returns>bool indicating whether the class is an Excel Worksheet</returns>
+        public static bool IsExcelWorksheet(IVBE ide, string projectName, string className)
+        {
+            using (var typeLibs = new VBETypeLibsAccessor(ide))
+            {
+                return IsExcelWorksheet(typeLibs.Get(projectName), className);
+            }
+        }
+
+        /// <summary>
+        /// Determines whether the specified document class is an Excel Worksheet
+        /// </summary>
+        /// <param name="project">Safe-com wrapper representing the VBA project</param>
+        /// <param name="className">Document class name, as declared in the VBA project</param>
+        /// <returns>bool indicating whether the class is an Excel Worksheet</returns>
+        public static bool IsExcelWorksheet(IVBProject project, string className)
+        {
+            using (var typeLib = TypeLibWrapper.FromVBProject(project))
+            {
+                return IsExcelWorksheet(typeLib, className);
+            }
+        }
+
+        /// <summary>
+        /// Determines whether the specified document class is an Excel Worksheet
+        /// </summary>
+        /// <param name="project">Safe-com wrapper representing the VBA component</param>
+        /// <returns>bool indicating whether the class is an Excel Worksheet</returns>
+        public static bool IsExcelWorksheet(IVBComponent component)
+        {
+            return IsExcelWorksheet(component.ParentProject, component.Name);
+        }
+
+        /// <summary>
+        /// Determines whether the specified document class is an Excel Worksheet
+        /// </summary>
+        /// <param name="projectTypeLib">Low-level ITypeLib wrapper representing the VBA project</param>
+        /// <param name="className">VBA Document class name, as declared in the VBA project</param>
+        /// <returns>bool indicating whether the class is an Excel Worksheet</returns>
+        public static bool IsExcelWorksheet(TypeLibWrapper projectTypeLib, string className)
+        {
+            return DoesClassImplementInterface(projectTypeLib, className, "Excel._Worksheet");
+        }
+
+        /// <summary>
+        /// Determines whether the specified document class is an Excel Workbook
+        /// </summary>
+        /// <param name="ide">Safe-com wrapper representing the VBE</param>
+        /// <param name="projectName">VBA Project name, as declared in the VBE</param>
+        /// <param name="className">Document class name, as declared in the VBA project</param>
+        /// <returns>bool indicating whether the class is an Access Form</returns>
+        public static bool IsAccessForm(IVBE ide, string projectName, string className)
```

Comment:
The XML doc says `Excel workbook` but this is for Access form. Other similar procedures have that, too so should be updated.